client/src/components/StarsInfoOpponent.tsx

```diff
@@ -0,0 +1,46 @@
+import filledStar from "../assets/FilledStar.svg";
+import unfilledStar from "../assets/UnfilledStar.svg";
+
+type StarsInfoOpponentProps = {
+  opponentName: string;
+  score: number;
+};
+
+const StarsInfoOpponent = ({ opponentName, score }: StarsInfoOpponentProps) => {
+  return (
+    <div className="fixed top-5 right-6 text-2xl text-white font-bold bg-[#FF5959] px-8 pt-1 pb-3 rounded-2xl">
+      <p>{opponentName}</p>
+
+      {score <= 0 && (
```

Comment:
Instead of copying and pasting all three stars, can we do a conditional inside each img. For example:

```
<img src={score >= 1 ? filledStar : unfilledStar} />
<img src={score >= 2 ? filledStar : unfilledStar} />
<img src={score >= 3 ? filledStar : unfilledStar} />
```

That way we don't need multiple copies for each case.